rtmp/src/main/java/net/ossrs/rtmp/SrsFlvMuxer.java

```diff
@@ -288,8 +292,28 @@ public void run() {
             }
 
             frame = mFlvVideoTagCache.poll(1, TimeUnit.MILLISECONDS);
-            if (frame != null) {
-              sendFlvTag(frame);
+            if (frame == null) {
+              if (lastSentIFrame != null) {
+                int diff = (int)(System.currentTimeMillis() - lastVideoFrameSentMs);
+                if (diff > 1000) {
+                  lastSentIFrame.dts = (int)(diff + lastVideoFrameSentDts);
```

Comment:
@SouOka 
下記のようにすれば、`lastVideoFrameSentDts `変数は不要そう
```suggestion
                  lastSentIFrame.dts = (int)(diff + lastSentIFrame.dts);
```

Reply:
@mn-D128 
313行目でもlastVideoFrameSentDtsを利用しており、指摘のようにするとそちらの方がnullになる可能性を孕んでしまうので、このままの方が安心かなと思います

Reply:
@SouOka 
OKです